### ISSUE TITLE
Course percentage for students misleading

### ISSUE THREAD

**Issue by julian-christl** [ls1intum/Artemis#3841]
### Describe the bug

If the course score is very close to 100% the course score gets rounded up.
However, in courses like EIST there is no rounding when it comes to the percentages when calculating the grade bonus. This can confuse students though.

### To Reproduce

Don't think this is required here.

### Expected behavior

Option A: Remove the percentage. Let the students draw their own conclusion
Option B: Leave the percentage but allow the course owners to set how this gets calculated
Option B: Leave it as it is. Then make clear that this percentage gets rounded and is not necessarily usable for grade calculations

### Screenshots

![Screen Shot 2021-07-27 at 00 32 27](https://user-images.githubusercontent.com/64264938/127068269-595a0870-1443-4a7e-849f-beaaabbb07e3.png)

### What browsers are you seeing the problem on?

Chrome

**Comment by krusche** [comment]
@julian-christl why do you think there is no rounding when it comes to calculating grades? Please correct me when I'm wrong, but I would assume that the course scores page would also use the value 100% in the shown example, so everything should be fine

**Comment by julian-christl** [comment]
@krusche In EIST we had [0.95; 1) => 9.5pts and [1; infty) => 10 pts
If you did round anyway in favour of the student it's probably more an issue with the slides that state it otherwise.
I was asked by a student whether he can rely on that percentage and according to the slides he can't. That's also what I told him.

**Comment by krusche** [comment]
> @krusche In EIST we had [0.95; 1) => 9.5pts and [1; infty) => 10 pts
> If you did round anyway in favour of the student it's probably more an issue with the slides that state it otherwise.
> I was asked by a student whether he can rely on that percentage and according to the slides he can't. That's also what I told him.

The slides do not state how we round ;-) and they also do not state which numbering system is used, so I don't think the slides are wrong, it's just a question how you interpret the slides. For the next year, we can add a note in the bottom of the slides that we use the default rounding here

**Comment by julian-christl** [comment]
Indeed. Since it was not defined, I assumed that there is no rounding at all since the mathematical definitions themselves exclude any rounding :D
But yes, specifying this on the slides would solve the confusion there for EIST.

@krusche I'd close it then since the feature is currently not required. Please close it if you think so yourself (since Johannes added a feature in #4071 I don't even think a documenation extension would be required)
